osquery/tables/system/npm_packages.cpp

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <boost/filesystem.hpp>
+
+#include <stdlib.h>
+#include <string>
+
+#include <osquery/core/tables.h>
+#include <osquery/filesystem/filesystem.h>
+#include <osquery/logger/logger.h>
+#include <osquery/utils/info/platform_type.h>
+#include <osquery/utils/json/json.h>
+#include <osquery/worker/ipc/platform_table_container_ipc.h>
+#include <osquery/worker/logging/glog/glog_logger.h>
+
+#ifdef WIN32
+#include "windows/registry.h"
+#endif
+
+namespace fs = boost::filesystem;
+
+namespace osquery {
+namespace tables {
+
+const std::set<std::string> kNodeModulesPath = {
```

Comment:
Consider initializing this std::set differently by checking which OS the code is running.

Reply:
For what it's worth, I think it's a reasonable pattern to initialize them all this way. We do it pretty commonly.

Reply:
@iko1 went with a simple `IFDEF` here for windows, I hope that is what you had in mind! Seph is right about POSIX, they would nearly be the exact same list (you can actually use homebrew on linux too if you like). So I think it's ok to group them together.